docs/data-masking-function-list.md

```diff
@@ -0,0 +1,877 @@
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------+
+    | mask_uk_nin('CT 26 46 83 D')       |
+    +------------------------------------+
+    | CT ** ** ** *                      |
+    +------------------------------------+
+    ```
+
+## mask_uuid(str [,mask_char])
+
+
+
+Masks a Universally Unique Identifier (UUID).
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| str | No | The string to be masked | String |
+| mask_char | Yes | The masking character | String |
+
+The `str` format is `********-****-****-****-************`.
+
+If the `mask_char` is not specified, the default is '*'. The `mask_char` value can be a multibyte character in any character set and may not be same character set as `str`.
+
+### Returns
+
+A string with the characters masked by a specified `mask_char` or that parameter's default value in the same character set as `str`.
+
+Returns an error if the length of `str` is incorrect.
+
+Returns `NULL` if `str`  is an incorrect length.
+
+### Example
+
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_uuid('9a3b642c-06c6-11ee-be56-0242ac120002');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------------------------------+
+    | mask_uuid('9a3b642c-06c6-11ee-be56-0242ac120002')     |
+    +-------------------------------------------------------+
+    |********_****_****_****_************                   |
+    +-------------------------------------------------------+
+    ```
+
+## masking_dictionary_remove(dictionary_name)
+
+
+
+Removes all of the terms and the dictionary. 
+
+Requires the `MASKING_DICTIONARIES_ADMIN` privilege.
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| dictionary_name | No | The dictionary to be removed | String |
+
+
+### Returns
+
+Returns a string value of `1` (one) if the operation is successful or `NULL` if the operation could not find the `dictionary_name`.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT masking_dictionary_remove('trees');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------------+
+    | masking_dictionary_remove('trees')       |
+    +------------------------------------------+
+    |                                        1 |
+    +------------------------------------------+
+    ```
+
+## masking_dictionary_term_add(dictionary_name, term_name)
+
+Adds the selected term to the dictionary. 
+
+Requires the `MASKING_DICTIONARIES_ADMIN` privilege.
+
+!!! note
+
+    The operation that adds the term to the dicitionary uses `INSERT IGNORE`. The `IGNORE` modifier has the following consequences:
+
+    * The inserted term is truncated without warnings if the term is longer than the mysql.masking_dictionaries.Dictionary or mysql.masking_dictionaries.Term maximum length.
```

Comment:
I would change "inserted term" to smth else because it can be confusing: this can be misread as if mysql.masking_dictionaries.Term will be truncated if mysql.masking_dictionaries.Dictionary >  maximum length of mysql.masking_dictionaries.Dictionary which is not the case. The Dictionary value is truncated if mysql.masking_dictionaries.Dictionary len > maximum length of mysql.masking_dictionaries.Dictionary, The Term is truncated if mysql.masking_dictionaries.Term len > maximum length of  mysql.masking_dictionaries.Term.